src/cmd/usb-util.js

```diff
@@ -404,6 +487,32 @@ async function openUsbDevices(args, { dfuMode = false } = {}){
 		});
 }
 
+
+/**
+* Waits for the device to reboot to reboot by checking if the device is ready to accept control requests.
+* It waits for a maximum of 60 seconds with a 1-second interval.
+*/
+async function waitForDeviceToReboot(deviceId) {
```

Comment:
Where is this function used?

Reply:
This was useful when I was working with dfu devices and flashing modules. This is again, used for an edge case on 6.1.1. I will remove this for now and insert it later if needed.